draft-pauly-intarea-proxy-config-pvd.md

```diff
@@ -137,15 +137,40 @@ match the hostname of the HTTP proxy. The "prefixes" array SHOULD be empty by de
 # Enumerating proxies within a PvD {#proxy-enumeration}
 
 This document defines a new PvD Additional Information key, `proxies`, that
-is an array of strings that is a list of proxy URIs (or URI templates
-{{!URITEMPLATE=RFC6570}}) that are available as part of a PvD. The new
-key is registered in {{iana}}.
-
-The kind of proxy is implied by the URI scheme and any template variables.
-For example, since UDP proxying {{CONNECTUDP}} has the URI template variables
-`target_host` and `target_port`, the URI
-"https://proxy.example.org:4443/masque{?target_host,target_port}" implies
-that the proxy supports UDP proxying.
+is an array of dictionaries, where each dictionary in the array defines
+a single proxy that is available as part of the PvD (see {{proxies-key-iana}}).
+Each proxy is defined by a proxy protocol, a proxy URI (or URI template
```

Comment:
I don't think we have ones that are just URIs, aren't they all templates? And to be pedantic all URIs are URI templates with zero parameters. So I'd just say URI template everywhere to make this simpler

Reply:
It seems clearer to me as URI or URI templates, since people may not be used to thinking of templates for socks, etc.

Reply:
Technically SOCKS doesn't use an URI, just a hostname and port. We could use a URI for SOCKS, but then we'd need to define that the URI can only have a host and port, no scheme, path or query

Reply:
Okay, adjusted to allow a hostname:port